src/evaluations/evaluations.module.ts

```diff
@@ -1,8 +1,9 @@
 import {Module} from '@nestjs/common';
 import {EvaluationsService} from './evaluations.service';
+import {EvaluationsController} from './evaluations.controller';
 
 @Module({
-  providers: [EvaluationsService],
+  providers: [EvaluationsService, EvaluationsController],
```

Comment:
This is wrong. Look at how the other modules specify the controller and/or refresh with the nestjs documentation on controllers and modules.